docs/install-run/install.rst

```diff
@@ -0,0 +1,316 @@
+
+==================================
+Installation
+==================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+Prerequisites
+=============
+
+CrateDB requires a `Java virtual machine`_ to run.
+
+- For CrateDB version 4.2 and newer
+   
+    - `Java virtual machine`_ is bundled with CrateDB and no extra installation is necessary.
+
+- For CrateDB version 3.0 to 4.1 
+   
+   - `Java 11`_ installation is required. We recommend using `Oracle's Java`_ on macOS and Microsoft Windows and OpenJDK_ on Linux Systems. Earlier versions required Java 8.
+
+
+
+Basic Installation
+===================
+
+For Linux, Micrsoft Windows and macOS
+
+#. Download `the latest CrateDB release`_.
+#. Once downloaded, expand the tarball
+
+    * using the sh 
+    
+    .. code-block:: sh
+
+       tar -xzf crate-*.tar.gz
+    
+    * using a tool like `7-Zip`_.
+  
+#. In the console change into the resulting crate directory
+
+    .. code-block:: sh
+
+       cd crate-*
+
+#. Run CrateDB as single instance bound to the local IP address
+
+   .. code-block:: sh
+
+      ./bin/crate
+
+#. You should be able to access it from your local machine by visiting ``http://localhost:4200/``
+
+#. With :kbd:`ctrl-c` you can stop CrateDB
+
+.. SEEALSO::
+
+      Consult the `CrateDB reference documentation`_ for help using this command.
+
+.. NOTE::
+
+      Other releases of CrateDB are `also available`_. Check out the `release notes`_ for specific information about each CrateDB release.
+   
+Next steps
+###########
+
+Now you have CrateDB up and running, :ref:`take the guided tour <use>`.
```

Comment:
@mfussenegger 
> This would replace the Quick Install On <XY> tutorials, right?
yes

Happy to proceed, after we have feedback from @mechanomi & @infoverload 🙂